modules/3d-tiles/src/lib/parsers/helpers/parse-3d-implicit-tiles.ts

```diff
@@ -224,13 +225,35 @@ export async function parseImplicitTiles(params: {
   return tile;
 }
 
-function getAvailabilityResult(availabilityData: Availability, index: number): boolean {
-  if ('constant' in availabilityData) {
-    return Boolean(availabilityData.constant);
+/**
+ * Check tile availability in the bitstream array
+ * @param availabilityData - tileAvailability / contentAvailability / childSubtreeAvailability object
+ * @param index - index in the bitstream array
+ * @returns
+ */
+function getAvailabilityResult(
+  availabilityData: Availability | Availability[],
+  index: number
+): boolean {
+  let availabilityObject: Availability;
+  if (Array.isArray(availabilityData)) {
+    /** TODO: we don't support `3DTILES_multiple_contents` extension at the moment.
```

Comment:
It would be nice to add a table to the docs with all the 3DTILES extensions and state whether we support or not and maybe some notes on how we support them.

Reply:
We have one for the tile-converter https://loaders.gl/docs/modules/tile-converter/cli-reference/supported-features